docs/source/app_developers_guide/docker.rst

```diff
@@ -2,696 +2,672 @@
+      ...
 
-After shutdown has completed, run this command:
+This terminal window will continue to display log messages as you run commands
+in other containers.
 
-.. code-block:: console
+.. note::
 
-  % docker-compose -f sawtooth-default.yaml down
+   If you need to reset the environment environment for any reason, see
+   "Stopping Sawtooth".
 
 
-Logging Into The Client Container
-=================================
+Step 4: Log Into the Client Container
+=====================================
 
-The client container is used to run Sawtooth commands, which is the usual
-way to interact with validators or validator networks.
+Sawtooth includes commands that act as a client application. The client
+container is used to run these Sawtooth commands, which interact with the
+validator through the REST API.
 
-Log into the client container by opening a new terminal window and
-running the following command. Note that ``sawtooth-shell-default`` specifies
-the client container name.
+To log into the client container, open a new terminal window and run the
+following command:
 
 .. code-block:: console
 
-  % docker exec -it sawtooth-shell-default bash
+  $ docker exec -it sawtooth-shell-default bash
+  root@75b380886502:/#
 
+In the following steps, the simplified prompt ``#`` is used for the commands
+that run from the terminal window for the client container. The ``$`` prompt
+denotes commands that should be executed from a terminal window on your host
+system. The full prompt ``root@21bb5af673a8:/#`` shows commands that run in the
+validator container.
 
-.. Important::
+.. important::
 
   Your environment is ready for experimenting with Sawtooth. However, any work
-  done in this environment will be lost once the container exits. The demo
-  Docker Compose file is useful as a starting point for the creation of your
-  own Docker-based development environment. In order to use it for app
-  development, you would need to take additional steps, such as mounting a host
-  directory into the container. See `Docker's documentation
-  <https://docs.docker.com/>`_ for details.
-
+  done in this environment will be lost once the container in which you ran
+  ``docker-compose`` exits. In order to use this demo environment for
+  application development, you would need to take additional steps, such as
+  mounting a host directory into the container. See the `Docker documentation
+  <https://docs.docker.com/>`_ for more information.
 
 .. _confirming-connectivity-docker-label:
 
-Confirming Connectivity
------------------------
+Step 5: Confirm Connectivity
+============================
 
-To confirm that a validator is running and reachable from the client
-container, run this ``curl`` command as root:
+Use these steps to be sure that you can connect to the validator
+and the REST API.
 
-.. code-block:: console
+ #. To confirm that a validator is running and reachable from the client
+    container, run this ``curl`` command:
```

Comment:
This is the same as below for checking connectivity to the REST API. Is there supposed to be a different command run here? Checking connectivity to the REST API with the /blocks route also checks that the validator is running, since the REST API depends on it to get a response.

Reply:
No, it's just that I didn't understand that checking connectivity to the REST API with the /blocks route also checks that the validator is running. I'll remove 3. and 4. from this step.